{"name": "Login with wrong email", "status": "passed", "description": "Signing in with the following credentials email : kamelmohsenkamel & password: Kimo2010", "attachments": [{"name": "SUCCESS: Login failed with incorrect credentials", "source": "4b898db8-b23b-462d-b047-37bbffd03435-attachment.png", "type": "image/png"}], "start": 1584959386433, "stop": 1584959396257, "uuid": "a2d511d7-4817-4e39-9862-f7d9636ca743", "historyId": "9aa68b1871d3b3962c9a317f03448176", "testCaseId": "edd1d0640244c5b16184751388881064", "fullName": "Web_Testing.Tests.test_signin.TestLogin#test_case_3", "labels": [{"name": "feature", "value": "Log in Tests"}, {"name": "subSuite", "value": "Login with wrong email"}, {"name": "parentSuite", "value": "End to End testing"}, {"name": "story", "value": "Failing login"}, {"name": "suite", "value": "Login Tests"}, {"name": "severity", "value": "blocker"}, {"name": "tag", "value": "Login"}, {"name": "tag", "value": "Do"}, {"name": "host", "value": "Kamel-388.local"}, {"name": "thread", "value": "12768-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Web_Testing.Tests.test_signin"}]}